### ISSUE TITLE
Option to search for continent without a country

### ISSUE THREAD

**Issue by FedeGueli** [GenSpectrum/cov-spectrum-website#390]
I think i would be useful if in future if there will be the option to exclude countries from Continent search.
It would be of great help to trace a variant outside countries with efficient genomic surveillance.

Thx

**Comment by corneliusroemer** [comment]
Yes, I can see this being useful, when you want to see spread of a variant
into surrounding countries.

Like for Alpha for example, Europe minus UK

Could be tricky to implement since countries are a simple selection at the
moment and this would require creating a query machinery for countries.

**Comment by FedeGueli** [comment]
Yes i could imagine it id hard to implement. thx very much for thid great tool.
